scripts-available/CDB_CartodbfyTable.sql

```diff
@@ -925,12 +925,15 @@ BEGIN
       AND t.typname = 'geometry'
       AND a.attnum > 0
       AND NOT a.attisdropped
-      AND (postgis_typmod_srid(a.atttypmod) > 0 OR srid.srid > 0)
+      AND ((postgis_typmod_srid(a.atttypmod) > 0 OR srid.srid > 0) OR (a.attname = const.geomcol))
       ORDER BY a.attnum
       LIMIT 1;
     
       IF FOUND THEN
         sql := sql || geom_transform_sql;
+      ELSE
+        RAISE NOTICE 'rtorre: spetial tricky one';
```

Comment:
I can change the string to not point to myself but yeah, having that trace can help me debug that other corner case if it appears :)

I'm thinking on another possible solution, just forcing the column types as it used to work.